app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,65 @@
+<% content_for :title, "#{@recipe.name}" %>
+
+<div class="row gap-3 my-5">
+  <div class="d-flex justify-content-between align-items-center">
+    <div class="recipe-main">
+      <h4>Preparation time: <%= @recipe.preparation_time %></h4>
+    </div>
+    <div class="recipe-sub">
+      <% if can?(:edit, @recipe) %>
+          <div class="form-group d-flex gap-3 align-items-center form-switch">
+            <%= button_to "#{@recipe.public ? 'Private' : 'Public'}", 
+                toggle_recipe_path(@recipe),
+                method: :patch,
+                class: "btn btn-link link-offset-2 link-underline link-underline-opacity-0" 
+              %>
+              <%= button_tag type: 'submit', class: "btn" do %>
+                <input type="checkbox" name="recipe[public]" class="form-check-input" <%= 'checked' if @recipe.public? %>>
+              <% end %>
+          </div>
+        <% end %>
+    </div>
+  </div>
+  <div class="recipe-main">
+    <h4>Cooking time: <%= @recipe.cooking_time %></h4>
+  </div>
+  <div class="recipe-main">
+    <p><%= @recipe.description %></p>
+  </div>
+</div>
+
+<div class="d-flex justify-content-between align-items-center">
+  <% if can?(:edit, @recipe) %>
+    <%= link_to "Generate Shopping List", shopping_list_path, class: 'btn btn-outline-secondary' %>
+  <% end %>
+  <% if can?(:edit, @recipe) %>
+    <a href="<%= new_recipe_recipes_food_path(@recipe) %>" class="btn btn-outline-secondary my-3">Add ingredient</a>
+  <% end %>
+</div>
+
+<hr class="my-3" />
+
+<div class="row my-5 gap-5">
+  <div class="col-md-12">
+    <table class="table table-striped table-bordered">
+      <thead>
+        <tr>
+          <th scope="col">#</th>
+          <th scope="col">Food</th>
+          <th scope="col">Quantity</th>
+          <th scope="col">Price</th>
+        </tr>
+      </thead>
+      <tbody>
+      <% @user_foods.each_with_index do |food, index| %>
+        <tr>
+          <th scope="row"><%= index + 1 %></th>
+          <td><%= food.name %></td>
+          <td><%= food.quantity %></td>
```

Comment:
- [Optional ] Though missed by the previous reviewer, it will be highly recommended to add the remove and modification buttons on this page as it is on the wireframe.
- This provides a more seamless experience for the user, and shows your adherence to requirements which you'll replicate for clients or your employer 👍 

### Your view:
![image](https://github.com/migo1/Recipie-App/assets/50990925/b5de9677-744c-4a12-8563-fcf25d77aaa6)

### Wireframe:
![image](https://github.com/migo1/Recipie-App/assets/50990925/bead6331-c24a-44fe-949f-16e3e98743b9)